docs/tutorials/analysing_runs.md

```diff
@@ -0,0 +1,138 @@
+visualizing the models, you can simply open the `report.html` file in a standard browser.
+
+## haddock3-traceback
+
+HADDOCK3 is highly customisable and modular, as the user can introduce several refinement, clustering, and scoring steps in a workflow.
+Quantifying the impact of the different modules is important while developing a novel docking protocol. The `haddock3-traceback` command
+is developed to assist the user in this task, as it allows to "connect" all the models generated in a HADDOCK3 workflow:
+
+```
+haddock3-traceback my-run-folder
+```
+
+`haddock3-traceback` creates a traceback subfolder within the `my-run-folder` directory, containing a `traceback.tsv` table:
+
+```
+00_topo1     00_topo2        01_rigidbody            01_rigidbody_rank       04_seletopclusts        04_seletopclusts_rank   06_flexref      06_flexref_rank 
+4G6K.psf     4I1B.psf        rigidbody_10.pdb        3                       cluster_1_model_1.pdb   1                       flexref_1.pdb   2       
+4G6K.psf     4I1B.psf        rigidbody_11.pdb        10                      cluster_1_model_2.pdb   3                       flexref_3.pdb   1       
+4G6K.psf     4I1B.psf        rigidbody_18.pdb        4                       cluster_2_model_1.pdb   2                       flexref_2.pdb   4      
+4G6K.psf     4I1B.psf        rigidbody_20.pdb        15                      cluster_2_model_2.pdb   4                       flexref_4.pdb   3       
+```
+
+In this table each row represents a model that has been produced by the workflow. The (typically) two used topologies are reported first,
+and then each module has its own column, containing the name and rank of the model at that stage. As an example, in the first row of the
+table above `rigidbody_10.pdb` is ranked 3rd at the `rigidbody` stage. Then, it becomes `cluster_1_model_1.pdb` (ranked 1st) after 
+the `seletopclusts` module. This model is then refined in `flexref_1.pdb`, which turns out to be the 2nd best model at the end of the workflow.
+
+The table can be easily parsed and used to evaluate the impact of different refinement steps on the different models.
+
+## The postprocess option
+
+You may want to run the `haddock3-analyse` and `haddock3-traceback` commands by default at the end of the workflow.
+The `postprocess` option of a standard HADDOCK3 configuration (.cfg) file is devoted to this task. At first, it forces HADDOCK3 
+to execute `haddock3-analyse` on all the `caprieval` folders found in the workflow, therefore loading data present in the CAPRI tables.
+Second, it executes the `haddock3-traceback` command.
+
+To activate this, just set `postprocess` to `true` at the beginning of your configuration file:
+
+```
+ ====================================================================
+# This is a HADDOCK3 configuration file
+
+# directory in which the docking will be done
+run_dir = "my-run-folder"
+
+# postprocess the run
+postprocess = true
```

Comment:
Open question (that should be discussed):
shoudn't the value of the `postprocess` parameter be true by default ?